src/glossary.js

```diff
@@ -208,6 +221,15 @@ Glossary.prototype.handleKeyup = function(e) {
   }
 };
 
+// Close glossary when clicking outside of glossa
+Glossary.prototype.closeOpenGlossary = function(e) {
+  if ( e.target !== this.toggleBtn && this.isOpen) {
+    if (!(closest(e.target, '#glossary'))) {
```

Comment:
Rather than hard-coding `#glossary`, let's use the value passed in with `this.selectors.glossaryID`.